### PR TITLE
Load json in back-end

### PR DESCRIPTION
**Pull request by janverhoeckx**
- modified end-point to serve fitness function data
- added JSON file reader

### CODE REVIEW

File: src/main/kotlin/dev/fff/fractalizer/FractalizerApplication.kt [hunk 1]
```diff
@@ -16,40 +13,10 @@ fun main(args: Array<String>) {
     runApplication<FractalizerApplication>(*args)
 }
 
-
-//FIXME  proxy fitness function to seperate package + renames?
 @RestController
-class Proxy {
-
-    private val httpClient = OkHttpClient()
+class Proxy(val fitnessFunctionDataLoader: FitnessFunctionDataLoader) {
```
Comment (evandenbeld) [inline src/main/kotlin/dev/fff/fractalizer/FractalizerApplication.kt]: Naam klopt niet meer

File: src/main/kotlin/dev/fff/fractalizer/dataloader/JsonFitnessFunctionLoader.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import dev.fff.fractalizer.model.FitnessFunction
+import org.slf4j.LoggerFactory
+
+@Service
+class JsonFitnessFunctionLoader(
```
Comment (evandenbeld) [inline src/main/kotlin/dev/fff/fractalizer/dataloader/JsonFitnessFunctionLoader.kt]: JsonFileFitnessFunctionLoader

File: src/main/kotlin/dev/fff/fractalizer/model/FitnessFunction.kt [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+import com.fasterxml.jackson.annotation.JsonInclude
+
+class FitnessFunction(
+        val name: String,
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+        val description: String?,
+        val okay: Boolean,
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+        val children: List<FitnessFunction>?
```
Comment (evandenbeld) [inline src/main/kotlin/dev/fff/fractalizer/model/FitnessFunction.kt]: map settings toevoegen

Comment (janverhoeckx) [reply src/main/kotlin/dev/fff/fractalizer/model/FitnessFunction.kt]: Will add it when we will actually use it.

File: src/main/kotlin/dev/fff/fractalizer/dataloader/JsonFitnessFunctionLoader.kt (hunk 2 above)
Comment (janverhoeckx) [reply src/main/kotlin/dev/fff/fractalizer/dataloader/JsonFitnessFunctionLoader.kt]: done

**Pull request merged**